apps/dotcom/client/package.json

```diff
@@ -67,18 +67,16 @@
 		"@types/node": "^22.15.31",
 		"@types/qrcode": "^1.5.5",
 		"@types/react": "^18.3.18",
-		"@vitejs/plugin-react-swc": "^3.7.2",
+		"@vitejs/plugin-react-swc": "^3.10.2",
 		"dotenv": "^16.4.7",
-		"eslint": "^9.19.0",
-		"eslint-plugin-formatjs": "^5.2.13",
 		"fast-glob": "^3.3.3",
 		"identity-obj-proxy": "^3.0.0",
 		"json5": "^2.2.3",
 		"kysely": "^0.27.5",
 		"lazyrepo": "0.0.0-alpha.27",
 		"pg": "^8.13.1",
 		"regexgen": "^1.3.0",
-		"vite": "^5.4.14",
+		"vite": "^7.0.1",
```

Comment:
decently big version jump. any upgrade concerns?

Reply:
nope - i tested both these out. 6.0 mostly bought a some new APIs around SSR that we don't use, and 7.0 mostly removed some deprecated stuff we don't use